syft/frameworks/torch/tensors/crypten/syft_crypten.py

```diff
@@ -0,0 +1,50 @@
+import torch
```

Comment:
Should we remove it from this PR if we don't use it?

Reply:
Yes, Done